CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ### (Next)
 * Your contribution here.
+* Update graphql-client github repository links in README - [@th1988]
```

Comment:
I'm not sure if this deserves a CHANGELOG entry, as there is no user-facing change in the code.

@dblock @ashkan18 Any thoughts?

Reply:
It's fine, but we do need to fix CI even if this is unrealated.